bot.js

```diff
@@ -40,6 +41,15 @@ function postCardImageTweet(status, cardName) {
 		.catch(e => logError('Failed to download image: ' + e));
 }
 
+function postSvgTweet(status, svgString, altText = "Rendered image") {
+	svg2png(new Buffer(svgString))
+		.catch(e => logError('Failed to create png: ' + e))
+		.then(data => twitter.uploadTwitterImageData(data.toString('base64')))
+		.catch(e => logError('Failed to upload image: ' + e))
+		.then(twitterImage => twitter.postImageTweet(twitterImage, altText, status))
+		.catch(e => logError('Failed to post tweet: ' + e));
```

Comment:
Can't we just catch once at the end and rely on the exception indicating what sort of error it was?